src/ngircd/ngircd.c

```diff
@@ -477,6 +491,9 @@ Show_Help( void )
 #endif
 	puts( "  -t, --configtest   read, validate and display configuration; then exit" );
 	puts( "  -V, --version      output version information and exit" );
+#ifdef SYSLOG
+	puts( "  -y, --syslog       log to syslog even when using -n" );
```

Comment:
`"  -y, --syslog       log to syslog even when running in the foreground (-n)"`?